instrumentation-tests/src/androidTest/java/com/mapbox/navigation/instrumentation_tests/utils/idling/RouteAlternativesIdlingResource.kt

```diff
@@ -52,14 +58,24 @@ class RouteAlternativesIdlingResource(
         alternatives: List<DirectionsRoute>,
         routerOrigin: RouterOrigin
     ) {
+        if (!onRoutesAlternatives.invoke(routeProgress, alternatives, routerOrigin)) {
```

Comment:
It's not really clear from the name `onRoutesAlternatives` when it returns true and when false. Because of that it's also not clear why you return here. Could you specify that in the name or at least in the docs?